includes/checkout/functions.php

```diff
@@ -414,3 +414,184 @@ function edd_enforced_ssl_asset_filter( $content ) {
 
 	return $content;
 }
+
+/**
+ * Given a number and algorithem, determine if we have a valid credit card format
+ *
+ * @since  2.4
+ * @param  integer $number The Credit Card Number to validate
+ * @param  string  $method The algorithem to use, by default our build in Luhn method
+ * @return bool            If the card number provided matches a specific format of a valid card
+ */
+function edd_validate_card_number_format( $number = 0, $method = 'luhn' ) {
```

Comment:
Originally when I wrote it, it ran a filter on it using the `$method` variable. We can remove it.